events/planner.py

```diff
@@ -1,43 +1,211 @@
+        message = self.get_message(meeting_creator_id)
+        send_mail_notification(email_title, message, meeting_creator_id.user_id.email)
+
+    def get_message(self, meeting_creator_id):
+        meeting_description = (
+            f"Meeting description: {self.event_id.description}\n" if self.event_id.description != "" else ""
+        )
+        meeting_location = f"Meeting location: {self.event_id.location}\n" if self.event_id.location != "" else ""
+        message = (f"Hi {meeting_creator_id.user_id.username}! The meeting results are here!!!"
+                   f"Meeting title: {self.event_id.title}\n"
+                   f"{meeting_description}"
+                   f"{meeting_location}"
+                   "The chosen meeting date is:\n"
+                   f"Starting date: {time_format(self.event_id.date_time_start)}\n"
+                   f"Ending date: {time_format(self.event_id.date_time_end)}\n"
+                   "The participant that will take part in the meeting:\n"
+                   f"{self.participants_names_who_can_meet}\n"
+                   "The participant that will not take part in the meeting:\n"
+                   f"{self.participants_names_who_cant_meet}\n")
+        return message
+
+    @staticmethod
+    def send_email_no_suitable_meeting_found(creator_id):
+        email_title = "No suitable meeting found"
+        message = "Unfortunately, no participant voted for the meeting and therefore no suitable meeting was found."
+        send_mail_notification(email_title, message, creator_id.user_id.email)
+
+    @staticmethod
+    def send_timeout_voting_notification_email_for_participants(message, meeting_creator_id):
+        meeting_id = meeting_creator_id.event_id
+        all_possible_meetings = OptionalMeetingDates.objects.get_all_event_dates(meeting_id)
+        all_meeting_participants = EventParticipant.objects.get_an_event_participants_without_creator(meeting_id)
+        email_title = "Voting for a meeting is about to end"
+        message = (f"The voting for the event {meeting_id.title} is about to end.\n"
+                   "Please finish your vote.\n"
+                   "If you did not vote for the meeting, you will not be able to attend.")
+        for participant in all_meeting_participants:
+            if not PossibleParticipant.objects.did_user_vote(all_possible_meetings, participant.user_id):
+                send_mail_notification(email_title, message, participant.user_id.email)
+                create_notification(message, participant)
+
+    @staticmethod
+    def calc_last_notification(voting_timeout):
+        minutes_before_timeout = 15
+        return voting_timeout - timedelta(minutes=minutes_before_timeout)
+
+    @staticmethod
+    def get_timeout(meeting_id):
+        def get_max_datetime(datetime1, datetime2):
+            if datetime1.date_time_start < datetime2.date_time_start:
+                return datetime1
+            return datetime2
```

Comment:
you can do it shorter like:
```suggestion
            return datetime1 if datetime1.date_time_start < datetime2.date_time_start else datetime2
```

Reply:
Done 👍